main.go

```diff
@@ -37,22 +37,29 @@ func main() {
 
 	var tags []string
 
-	switch repo {
-	case "hub.docker.com":
+	switch {
+	case strings.Contains(repo, "hub.docker.com"):
 		t, err := retrieveFromDockerHub(image)
 		if err != nil {
 			fmt.Fprintln(os.Stderr, err)
 			os.Exit(1)
 		}
 
 		tags = t
-	case "quay.io":
+	case strings.Contains(repo, "quay.io"):
 		t, err := retriveFromQuay(image)
 		if err != nil {
 			fmt.Fprintln(os.Stderr, err)
 			os.Exit(1)
 		}
 
+		tags = t
+	case strings.Contains(repo, "amazonaws.com"):
```

Comment:
Is `strings.HasSuffix(repo, "amazon.com")` better than this?

The ECR URI format is as follows
```
[account id].dkr.ecr.[region name].amazonaws.com/[repository name]
```

```
```

Reply:
`amazonaws.com` is okay.

But this case condition should be `strings.HasSuffix`.

```suggestion
	case strings.HasSuffix(repo, "amazonaws.com"):
```

Reply:
fixed!